resources/RESOURCES.md

```diff
@@ -78,6 +78,7 @@ client.on('debug', console.log)
 _ _
 **Affordable hosting providers:**
 • [OVH](https://www.ovh.com/us/vps/) Starting at $3.35/mo for 1 core, 2GB RAM, 20GB SSD
+• [Contabo](https://contabo.com/en/vps/) Starting at €5.99/mo for 4 core, 8GB RAM, 200GB SSD
```

Comment:
"Cloud VPS S", which i assume you are basing this on here, would be $8.49 according to <https://contabo.com/en>
not a fan of mixing currencies here